Language.md

```diff
@@ -1 +1,74 @@
+So they must be doing something right.
+
+So getting right down to the facts. Some of the things I find really useful about JavaScript:
+
+**1. It's made for the web**
+
+More like the web is made of JavaScript.
+**Literally**.Almost every website you come across has JavaScript in some way or form. In the early days when the Java guys were trying to get their own version of the web started (*applets*,ahem ahem).
+
+A relatively unknown language called EcmaScript was born. It had it's quirks and bugs for sure. Poor design decisions? Absolutely .But it took a hold of the programmers like no other.
+The brilliant thing about JavaScript was it was designed from the ground up with the internet in mind ,which sadly Java did not as they tried to shoehorn this extra 'web functionality' into their admittedly admirable language (I like all things Java, as you might have guessed)
+
+**2. It's evolving like crazy**
+
+JavaScript is changing at a rapid pace, what was cool six months ago has been replaced by a leaner and meaner counterpart. *Gulp* replaces *Grunt*.*Socket.io* replaces *Web-RTC* and so on and so forth.
+
+This is what gives Js an edge over it's counterparts. The team behind it is dedicated and focused and this has led to the developers to take up arms and join the fight at bettering JavaScript.
+The amazing community behind it is one of the reasons behind it's success.
+
+**3. Async FTW**
+
+For those who don't know, Async or *asynchronous* support is the ability for a client to request something and go on with it's other duties ,meanwhile leaving a guard or a watch keeper in place to constantly look whether the request has received a response. Upon receiving the response the program can respond accordingly.
+
+But the important thing here is that the program doesn't have to **wait** for the response.
+It can do other important things and deal with the outcome when it is ready to grace us with it's presence.
+
+This has unbelievable advantages(and disadvantages,*callback hell is a terrible place to be in ,more on that later*)
+
+* The primary advantage is increased responsiveness and the illusion of speed, which is very important for a web developer.
+
+* The second advantage can be engaging in long lasting database interactions(background tasks)
+
+Async support is uncommmon in a lot of languages (though it is picking up steam),JavaScript is pretty ahead of the game on this front.
+
+**4. Server Side A.K.A Node.js**
+
+In the early days JavaScript was really limited, which means it was only useful on the client side. The recent phenomenon of node.js has changed this scenario rapidly. Node.js can frankly add a backend to your frontend using only JavaScript, Another way of looking at it is ,it can create a server.
+No PHP, No Python needed.
+
+This has led to applications being created only in one language. So we can interact with the database, handle client requests all using Javascript.
```

Comment:
Mention about Universal JavaScript or Isomorphic JavaScript. Do link to nice medium articles for the same